packages/config/css/animations.css

```
@@ -0,0 +1,298 @@
+@keyframes panelSlideLeftIn {
+  0% {
+    transform: translate-x-0;
+    opacity: 1;
+  }
+  100% {
+    transform: translateX(-100%);
+    opacity: 0;
+  }
+}
+
+@keyframes panelSlideRightOut {
```

Stylelint (17.11.0)

[error] Expected keyframe name "panelSlideRightOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)